internal/fuzzbinder/fuzzscraper.go

```diff
@@ -0,0 +1,335 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package fuzzbinder
+
+// This file provides the scraper module that helps to get the fuzzing statistics
+// from the ClusterFuzz and OSS-Fuzz reports.
+
+import (
+	"bufio"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"strconv"
+	"strings"
+
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+
+	"cloud.google.com/go/storage"
+	"google.golang.org/api/iterator"
+)
+
+// CoverageBucket is the OSS-Fuzz Google Cloud Storage bucket containing
+// the coverage reports.
+const CoverageBucket = "oss-fuzz-coverage"
+
+// OSS-Fuzz uses Clang to generate coverage summary reports.The coverage
+// summary contains fine-grained data related to coverage metrics.
+// CoverageSummary contains the coverage report summary.
+type CoverageSummary struct {
+	Data    []CoverageSummaryData `json:"data"`
+	Type    interface{}           `json:"type"`
```

Comment:
Why is the type `interface{}`? 
Also since we use go1.18, you can use `any` (Ref go/gocomments#use-any). 
We have `interface{}` in the slsa (`v02`) package but that is because that code is copied from the slsa repository.

Reply:
The type is `interface{}` because I am not extracting data in `Type` or `Version` or `Files` . In other words, I only define the types of the parts on which the extracted coverage data depends. For the other parts, I don't want to define type constraints that might generate errors in the future if their type or structure changes, since these changes will not impact the final extracted coverage data.

I used `interface{}` for consistency but updated it to `any` as suggested.